packages/runtime/src/widgets/Button.tsx

```diff
@@ -74,7 +74,13 @@ export const Button: React.FC<ButtonProps> = ({ id, onTap, ...rest }) => {
         }}
       >
         {!loading && (
-          <>
+          <div
+            style={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
```

Comment:
Hmm this will break how styling will work by inserting another child element. These styles are already on the parent element - did this regress?

Reply:
Yea. The label and icons are not aligned since this [PR](https://github.com/EnsembleUI/ensemble-react/pull/572) there's a span containerizing the icons and label, we need to align them

Reply:
Weird I didn't notice while testing. Not sure how this fixes the issue if that span is still wrapping the icons and label?